engine/src/main/java/org/terasology/rendering/dag/stateChanges/DisableCapability.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.stateChanges;
+
+import org.terasology.rendering.dag.RenderPipelineTask;
+import org.terasology.rendering.dag.StateChange;
+import org.terasology.rendering.dag.tasks.DisableCapabilityTask;
+
+/**
+ * TODO: Add javadocs
+ */
+public class DisableCapability implements StateChange {
```

Comment:
I am not quite convinced about this DisableCapability/EnableCapability pattern. I think each capability that can be enabled/disabled should have a single state change, like we did for wireframe. If the capability is enabled by default we can call the class "Disable<Something>". If the capability is disabled by default we can call the class "Enable<Something>". Which makes me think by the way SetWireframe should probably be called "EnableWireframe". But we can leave it as it is for now.